source/install/self-managed-changelog.md

```diff
@@ -5,11 +5,120 @@
+ - Added a ``threadsOnly`` query parameter for getting user threads.
+ - The Elasticsearch indexing job is resumable now. Stopping a server while the job is running will put the job in pending status and will resume the job when the server starts. The job can still be explicitly canceled via the **System Console**.
+ - Added a new stats field under the channel object for graphql.
+ - Added a new “License" button in **System Console > Edition and License**.
+
+### Bug Fixes
+ - Fixed an issue where permalinks to direct and group message posts did not show a preview.
+ - Fixed an issue when Collapsed Reply Threads are enabled where marking a root post with a mention as unread displayed both a mention badge and the thread item being bolded.
+ - Fixed an issue where the public link to generate the API was getting called even if public links were disabled.
+ - Fixed an issue with onboarding page view events.
+ - Fixed an issue where the custom emoji **Next** button was out of view when a banner was present.
+ - Fixed an issue where it would appear that a user had a negative number of unread threads.
+ - Fixed an issue where marking the last post in a thread as unread didn't mark the thread as unread.
+ - Restored the rendering of main menu items from plugins in non-mobile view.
+ - Fixed the overflow of text in **Manage Channel Members** modal title.
+ - Fixed an issue where pagination was broken in **System Console > Groups**.
+ - Fixed an issue where thread updates did not show correctly after the computer woke up.
+ - Fixed an issue where a negative unread count sometimes appeared with Collapsed Reply Threads enabled.
+ - Fixed an issue where the modal to create a Custom Group got closed when pressing ENTER.
+ - Fixed an issue where group mention did not get highlighted in Professional License.
+   
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``EmailSettings`` in ``config.json``:
+    - Added ``EnableInactivityEmail`` setting to be able to disable inactive server email notifications.
+ - Under ``JobSettings`` in ``config.json``:
+    - Added a new cleanup job to regularly remove outdated config entries from the database. The threshold for this setting can be adjusted with ``CleanupConfigThresholdDays``.
+ - Under ``ElasticsearchSettings`` in ``config.json``:
+    - Elasticsearch and Bleve indexing have been revamped to be much more efficient and faster. The config parameter ``BulkIndexingTimeWindowSeconds`` for both Elasticsearch and bleve is now deprecated and no longer used. A new config parameter called ``BatchSize`` has been introduced instead. This parameter controls the number of objects that can be indexed in a single batch. This makes things more efficient and maintains a constant workload.
```

Comment:
Enterprise Edition only?

Reply:
Pinging @agnivade to confirm

Reply:
Elasticsearch is in enterprise. Bleve is in open source.